src/textual/widgets/_loading_indicator.py

```diff
@@ -22,8 +24,49 @@ class LoadingIndicator(Widget):
         content-align: center middle;
         color: $accent;
     }
+    LoadingIndicator.-overlay {
+        overlay: screen;
+        background: $boost;
+    }
     """
 
+    def apply(self, widget: Widget) -> AwaitMount:
```

Comment:
I actually expected `apply` to also be a `classmethod`.
Is there any reason why we prefer it not to be a `classmethod`?
Is it so that we can have greater control over which loading indicator gets overlayed on the given widget?

Reply:
It's so the caller can construct the LoadingIndicator.